cmd/util.go

```diff
@@ -304,21 +314,33 @@ func readProject() (*workspace.Project, string, error) {
 	// Now that we got here, we have a path, so we will try to load it.
 	path, err := workspace.DetectProjectPathFrom(pwd)
 	if err != nil {
-		return nil, "", errors.Wrapf(err,
-			"could not locate Pulumi.yaml project file (searching upwards from %s)", pwd)
+		return nil, "", errors.Wrapf(err, "Failed to find current Pulumi project because of "+
```

Comment:
Wrapped error messages should start with lowercase letter (by our and general Go convention).

I also don't understand why the new message is preferred over the old one - the words it adds don't seem to add any clarity?  I honestly prefer the former in isolation.